windows-cluster/variables.tf

```diff
@@ -0,0 +1,85 @@
+variable "name" {
+  type        = string
+  description = "The name of the cluster you would like to provision."
+}
+
+variable "nodes" {
+  type        = list(any)
+  description = "The pools of nodes that you would like to provision for this cluster."
+  default = [
+    {
+      name     = "server"
+      image    = "linux"
+      roles    = ["controlplane", "etcd", "worker"]
+      replicas = 1
+    },
+    {
+      name     = "windows"
+      image    = "windows"
+      roles    = ["worker"]
+      replicas = 1
+    }
+  ]
+}
+
+variable "infrastructure" {
+  type = object({
+    azure = optional(object({
+      enabled  = optional(bool, false)
+      location = optional(string, "West US")
+    }))
+  })
+  description = "Additional options to be provided to a supported cloud provider."
+  default = {
+    azure = {
+      enabled = true
+    }
+  }
+}
+
+variable "additional_servers" {
+  type        = list(any)
+  description = "Additional servers that should be created alongside your cluster. These servers will automatically be added to the same network as the cluster."
+  default     = []
+}
+
+variable "fleet_workspace" {
+  type        = string
```

Comment:
Is there value in letting this be configurable? IIRC it will always have to be fleet-default, and changing it will break things

Reply:
In case we need to test something related to Windows x Fleet when deployed into the non-default namespace?